algebird-test/src/test/scala/com/twitter/algebird/BatchedTest.scala

```diff
@@ -43,10 +44,10 @@ class BatchedLaws extends CheckProperties with Matchers with PropertyChecks {
   testBatchedMonoid[BigInt]("BigInt", 10)
   testBatchedMonoid[BigInt]("BigInt", 100)
   testBatchedMonoid[BigInt]("BigInt", 1000000)
-  testBatchedMonoid[BigDecimal]("BigDecimal", 1)
-  testBatchedMonoid[BigDecimal]("BigDecimal", 10)
-  testBatchedMonoid[BigDecimal]("BigDecimal", 100)
-  testBatchedMonoid[BigDecimal]("BigDecimal", 1000000)
+  testBatchedMonoid[BigDecimal]("BigDecimal", 1)(arbReasonableBigDecimals, implicitly[Monoid[BigDecimal]])
```

Comment:
I don't think you should need this. I think the issue is that you have the `Arbitrary` both implicit in BaseProperties imported as well as arbReasonableBigDecimals.

I'd rather not make the `BaseProperties` one implicit. That should fix the issue. That makes it opt in, as you did here, but I think that is more conservative since we don't "own" either `Arbitrary` or `BigDecimal`.